How to install

vojtechsokol opened the pull request.

GAZDOWN commented on installation.md:
```diff
@@ -0,0 +1,78 @@
+
+* [leappwf](https://github.com/leapp-to/leappwf)
+
+    **TODO**
```
IMO this should not be visible for outsiders. We can keep the track somewhere else, like i.e. in the comments at the beginning of the file, but not in the rendered output.

vinzenz replied:
leappwf is kind of obsolete at this point

GAZDOWN commented on installation.md:
```diff
@@ -0,0 +1,78 @@
+
+* [actor-browser](https://github.com/leapp-to/actor-browser)
+
+	**TODO**
```
ditto

vinzenz replied:
it might be useful for discoverablity - anyway not to interesting for the installation at this point though

GAZDOWN commented on installation.md:
```diff
@@ -0,0 +1,78 @@
+
+* [actor-sdk](https://github.com/leapp-to/actor-sdk)
+
+	**TODO**
```
ditto

vinzenz replied:
no not really - The sdk might be interesting however we have to define that, it's an early WIP

vinzenz commented on installation.md:
````diff
@@ -0,0 +1,71 @@
+* Prerequisites
+    * Enable copr repository
+        ```shell
+        sudo dnf install dnf-plugins-core -y
````
TBH this is actually not the right place to use anymore and inconsistent with moth of the things you explain to install via git below. 

leapp-tool on top of that is quite outdated but the last stable release and we are now trying to get to get leappctl to become the new CLI interface so we would have to focus on that there.

## leapp-tool
I recommend you to follow this for the last stable environment:

https://github.com/leapp-to/leapp/blob/2018-backup/docs/source/getstarted.rst

## leappctl
We should start using https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/ as a source

GAZDOWN replied:
True, I would suggest to have two approaches, one described here will be how to use rpms. The other, on the same page will be a short mention of dev-env where there is thoroughly described how to use automake with our project. This will give the user ways how to use the project on RPM based distros as well as any other, if he/she would like to.

vojtechsokol commented:
Changed the instructions to use the `group_leapp` copr repository and added link to devenv.
The current rendering can be seen [here](https://vojtechsokol.github.io/installation).

Also should I mention somewhere warning that Leapp is right now in kind of pre-alpha state?

GAZDOWN commented on installation.md:
````diff
@@ -0,0 +1,31 @@
+## CentOS 7
+
+	```shell
+	sudo yum install yum-utils -y
````
Move -y parameter behind the positional install options. It makes it more readable.

GAZDOWN commented on installation.md:
````diff
@@ -0,0 +1,31 @@
+
+## Fedora 26 and 27
+
+	```shell
````
According to the render, this does not work. I believe that the space/tab in front of ``` is not allowed. Also, when you use it this way - empty line before and after the block together with the indentation, it is considered automatically as a block.
